components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ImportUtils.java

```diff
@@ -237,7 +237,8 @@ public static API importApi(String extractedFolderPath, APIDTO importedApiDTO, B
                 }
                 targetApi.setOrganization(organization);
                 importedApi = PublisherCommonUtils
-                        .updateApi(targetApi, importedApiDTO, RestApiCommonUtil.getLoggedInUserProvider(), tokenScopes);
+                        .updateApi(targetApi, importedApiDTO, RestApiCommonUtil.getLoggedInUserProvider(), tokenScopes,
+                                organization);
             } else {
```

Comment:
No need to pass organization . we could use targetApi.getOrganization()

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10621/commits/d711021b49bdfd26cece08490dee2dcd69b5d626